src/styles/navbar.module.css

```diff
@@ -0,0 +1,29 @@
+@import "./palette.css";
+@import url("https://fonts.googleapis.com/css?family=Roboto&display=swap");
+
+.logo {
+  width: 10rem;
+  height: 2rem;
+}
+
+.navbar {
+  background-color: var(--color1);
+  font-family: "Roboto", sans-serif;
+}
+
+.navLink {
```

Comment:
suggesting the following styling:
```css
.navLink {
  color: var(--white) !important;
  font-size: 0.78rem;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  border-bottom: 1px transparent solid;
}
```
- the overflow related properties are just a safeguard so that the links never get multiline
- the border property removes the derpy behaviour of adding/removing border. the color is changed on hover